boards/common/arduino-atmega/include/arduino_board.h

```diff
@@ -24,6 +24,8 @@
 
 #include "arduino_pinmap.h"
 
```

Comment:
Might be bit nitpicking, there should be no extra newline between includes as in other `arduino_boar.h` files.

Reply:
Fixed and directly squashed